controllers/chat/moveToAnon.js

```diff
@@ -5,6 +5,7 @@ const {responseError, responseSuccess} = require('../../utils/Responses');
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const UsersFunction = require('../../databases/functions/users');
 const Getstream = require('../../vendor/getstream');
+const BetterSocialConstantListUtils = require('../../services/bettersocial/constantList/utils');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new module `BetterSocialConstantListUtils` is imported but there's no check to ensure that the module is available or not. If the module is missing, it will throw an error and halt the execution of the script. It would be better to handle this potential issue.

```diff
+ try {
+   const BetterSocialConstantListUtils = require('../../services/bettersocial/constantList/utils');
+ } catch (error) {
+   console.error('Failed to import module BetterSocialConstantListUtils:', error);
+   return;
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->